src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1232,6 +1239,37 @@ def _get_log(url, user_name, password, log_file=None):
                 if not data:
                     break
                 f.write(data)
+    elif number_of_lines:  # print last 'n' number of lines
+        api_str = r.read().decode(encoding='utf-8')
+
+        href_count = api_str.count('https')
+
+        if href_count > 1:
+            logger.info('Will show last %s lines of logs per instance', number_of_lines)
+
+        for x in range(0, href_count):  # pylint: disable=unused-variable
+            index_of_path = api_str.index('\",\"path')
+            href_str = api_str[api_str.index("https"):index_of_path]
+            size = api_str[api_str.index('\"size\":') + len('\"size\":'):api_str.index(',\"href\"')]
```

Comment:
Could you please paste here a sample `api_str` so I know what it looks like?

Reply:
Here is the api_str for my site that has 3 instances.

'[{"machineName":"RD00155DFE584B","lastUpdated":"2017-11-07T19:26:12Z","size":652,"href":"https://yeeboi.scm.azurewebsites.net/api/vfs/LogFiles/2017_11_07_RD00155DFE584B_docker.log","path":"/home/LogFiles/2017_11_07_RD00155DFE584B_docker.log"},{"machineName":"RD00155DFE6403","lastUpdated":"2017-11-07T19:26:11Z","size":652,"href":"https://yeeboi.scm.azurewebsites.net/api/vfs/LogFiles/2017_11_07_RD00155DFE6403_docker.log","path":"/home/LogFiles/2017_11_07_RD00155DFE6403_docker.log"},{"machineName":"RD00155DFE6800","lastUpdated":"2017-11-07T19:26:11Z","size":652,"href":"https://yeeboi.scm.azurewebsites.net/api/vfs/LogFiles/2017_11_07_RD00155DFE6800_docker.log","path":"/home/LogFiles/2017_11_07_RD00155DFE6800_docker.log"}]'